phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/PhoenixConfigurationUtil.java

```diff
@@ -290,6 +291,18 @@ public static void setOutputCluster(final Configuration configuration,
     public static Class<?> getInputClass(final Configuration configuration) {
         return configuration.getClass(INPUT_CLASS, NullDBWritable.class);
     }
+
+    public static boolean getCustomTest(final Configuration configuration) {
```

Comment:
Likewise, this is letting test details leak into production code a bit too much.